eventlogger/src/test/java/gov/cms/ab2d/eventlogger/eventloggers/sql/AllMapperEventTest.java

```diff
@@ -48,11 +48,27 @@ void exceptionApiRequestTests() {
                 new ApiRequestEventMapper(null).log(new ErrorEvent()));
     }
 
+    @Test
+    void createError() {
+        // There is no mapping. Make sure it doesn't create an issue
+        BeneficiarySearchEvent bene = new BeneficiarySearchEvent();
+        sqlEventLogger.log(bene);
+        sqlEventLogger.updateAwsId("TEST", bene);
```

Comment:
Is the idea with this test and the one below to see if it doesn't throw an exception?

Reply:
Yes, even though it doesn't have an sql mapping. I don't want logging to mess anything up.